i18n/en.yml

```
@@ -153,13 +153,15 @@ error_fetch_top_10_platforms: "Failed to fetch top 10 platforms data"
 error_fetch_top_10_platforms_user: "Failed to fetch top 10 platforms data for user {user_id}"
 error_fetch_top_10_users: "Failed to fetch top 10 users data"
 error_fetching_data: "Error fetching data from Tautulli API"
+error_fetching_data: "Error fetching data from Tautulli API"
```

yamllint

[error] duplication of key "error_fetching_data" in mapping

(key-duplicates)